packages/circle-demo-webapp/app/routes/api.createWallet.tsx

```diff
@@ -64,7 +60,7 @@ export async function action({ request }: ActionFunctionArgs) {
       ],
     });
 
-    return successResponse('Success');
+    return Response.json(res.data?.wallets[0]);
```

Comment:
cool, we return the wallet that we just created to display it in the success screen